src/com/facebook/buck/swift/SwiftCompileBase.java

```diff
@@ -167,6 +167,10 @@
   
   @AddToRuleKey private final boolean indexWhileBuilding;
 
+  @AddToRuleKey private final boolean localIndexStore;
+
+  @AddToRuleKey private final Optional<String> numberOfThreads;
```

Comment:
This property shouldn't be added to the RuleKey calculation because that it's only the number of threads to use, this will not affect artifacts.